### DISCUSSION TITLE
Phalcon 5 & MongoDB

### DISCUSSION THREAD

**Discussion by gafsou20** [phalcon/phalcon#423] [General]
Hello everyone, 
I have been using Phalcon 3.4 with mongodb for over 4 years now.  I find that it makes a shocking duo, between the performance of phalcon and the storage of mongo.  I started to upgrade my code with version 5 of Phalcon, I managed to recreate my micro framework environment but I am stuck with the mongodb integration.  I can't find a compatible adapter.  
Has anyone faced the same situation?

**Comment by Jeckerson** [comment]
Hello

Here is Phalcon v5 compatible package for MongoDB - https://github.com/phalcon/incubator-mongodb
In case any problem - please submit an issue on central `phalcon/incubator` repository.
